lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -224,7 +226,7 @@ def ssl_options
     end
 
     def http_compression
-      client_settings.fetch(:http_compression, {})
+      client_settings.fetch(:http_compression, false)
```

Comment:
this is a copy paste mistake. it is not necessary for the PR, but it was making some code paths use http_compression when it shouldn't, since `{}` was being used instead of false.